### ISSUE TITLE
discuss quarto in appendix B

### ISSUE THREAD

**Issue by hawkrobe** [langcog/experimentology#110]
Appendix B currently focuses on RMarkdown, but my understanding is that it’s slowly being phased out in favor of [Quarto](https://quarto.org/). 

Rmd isn't necessarily going to be deprecated anytime soon (e.g. at least according to [this FAQ](https://quarto.org/docs/faq/rmarkdown.html) and this [blog post](https://www.apreshill.com/blog/2022-04-we-dont-talk-about-quarto/)), but developer energy has shifted. Additionally, it’s the default being pushed in the most recent versions of RStudio, so newer users may be confused about how to get the 'old' RMarkdown version we're talking about. 

It may be worth (minimally) making a box or (maximally) revamping that appendix to ‘future-proof’ it a bit to be less confusing? I made the switch to quarto a couple months ago and have found the syntax/features to be a lot more intuitive in some ways that could simplify the presentation.

**Comment by mcfrank** [comment]
Yes, great idea.. I haven't adopted Quarto yet but perhaps we should...
